imsv-docs-astro/src/content/docs/guides/core-concepts/card-payments-101.mdoc

```diff
@@ -34,13 +34,17 @@ have not yet cleared appear to the cardholder as “pending” transactions.
 A Clearing message indicates finalization of a payment. In an e-commerce
 setting, this often happens after order fulfillment. A clearing message may
 have a value that is greater or less than the authorized amount. Authorized
-funds are not settled until cleared. Authorizations expire when not cleared
-within a month. Cardholders regain full control of funds held by expired
-Authorizations.
+funds are not settled until cleared.
+
+Authorizations expire when not cleared. The expiry period is between 7 and 30
+days dependent upon the particulars of the Authorization. Cardholders regain
+full control of funds held by expired Authorizations.
+
 
 ## Settlement
 
-All cleared transactions need to be paid out to Mastercard so that merchants
-can receive payment. The process of paying out cleared transactions is referred
-to as Settlement. Settlements are batched daily or at a cadence that is
-appropriate based on the current transaction volume.
+All cleared payment transactions need to be paid to Mastercard by Immersve. The
+process of paying out cleared transactions is referred to as Settlement.
+Settlements are are conducted by Immersve at a cadence as instructed by
+Mastercard. An Immersve partner organisation does not actively participate in,
+or have visibility of, the settlement process.
```

Comment:
```suggestion
Mastercard. An Immersve partner organisation does not actively participate in the settlement process or has visibility of it.
```
I'd rewrite it with less commas but Hemingway Editor complains about my version too so I'm not sure what's best here. Maybe we don't need to mention visibility and just say that partner org is not participating?